src/expr/proof_rule.h

```diff
@@ -701,12 +701,72 @@ enum class PfRule : uint32_t
   ARRAYS_TRUST,
 
   //================================================= Bit-Vector rules
+  // Note: bitblast() represents the result of the bit-blasted term as a
+  //       bit-vector consisting of the output bits of the bit-blasted circuit
+  //       representation of the term
```

Comment:
Reference the method in the code that has the semantics of this function.

Reply:
There is no code, just a kind.